app/controllers/ops_controller/ops_rbac.rb

```diff
@@ -1062,7 +1094,9 @@ def rbac_group_get_form_vars
 
     if params[:check]                               # User checked/unchecked a tree node
       if params[:tree_typ] == "tags"                # MyCompany tag checked
-        cat, tag = params[:id].split('cl-').last.split("_xx-") # Get the category and tag
+        # cat, tag = params[:id].split('cl-').last.split("_xx-") # Get the category and tag
+        cat = params[:cat]
+        tag = params[:val]
```

Comment:
```ruby
cat, tag = params.values_at(:cat, :val)
```

Reply:
But I'd just simply use the `params[:cat]` and `params[:tag]` in the code below, no need for allocation if you don't use it elsewhere.